### DISCUSSION TITLE
HC_MISC/memory/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#65] [Show and tell]
# HC_MISC/memory/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://ctf.tj.cn/HC_MISC/memory/

**Comment by Tlenzer** [comment]
太细辣太细辣，大佬什么时候可以顺便把磁盘取证做了555

**Comment by ZGC1314** [comment]
还有插件codetective：https://github.com/blackthorne/Codetective。
